src/udev/udev-builtin-net_id.c

```diff
@@ -359,9 +359,9 @@ static int dev_pci_slot(sd_device *dev, struct netnames *names) {
                 if (naming_scheme_has(NAMING_SLOT_FUNCTION_ID) &&
                     sd_device_get_sysattr_value(hotplug_slot_dev, "function_id", &attr) >= 0) {
                         _cleanup_free_ char *str = NULL;
-                        int function_id;
+                        uint32_t function_id;
 
-                        if (safe_atoi(attr, &function_id) >= 0 &&
+                        if (safe_atou32(attr, &function_id) >= 0 &&
```

Comment:
Hmm, but do we actually care what the kernel uses *internally*, since this data is passed to us a string? Shouldn't we actually use safe_atou64 here, to be future-proof in case the kernel one day expands to a wider range?
(Esp. that we then *use* the variable in snprintf with `%08x`…)